src/Containers/Popup.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import styled from 'styled-components';
+import { LabelLayoutProps } from '@Layouts';
+
+export interface PopupProps extends LabelLayoutProps {
+    left?: number;
+    top?: number;
+    width?: number;
+    height?: number;
+    popup?: boolean;
+}
+
+export const Popup: React.FC<PopupProps> = ({
+    children,
+    left,
+    top,
+    width,
+    height,
+    popup,
+    ...props
+}): React.ReactElement => {
+    return (
+        <Container popup={popup} left={left} top={top}>
```

Comment:
I don't think pop-up is a standard div attribute can u clarify what this does as it doesn't look like it does anything (I tried passing in false in your story and it just changed nothing)

Reply:
Oh it's in popup.tsx
Due to the changes I made to slider.tsx this is useless now thank thank you for pointing it out